deploy-tool/main.go

```diff
@@ -85,17 +85,34 @@ type TerraformOutput struct {
 	} `json:"specific_manifests"`
 }
 
+type privateKeyMarshaler func(*ecdsa.PrivateKey) ([]byte, error)
+
+// marshalX962UncompresedPrivateKey encodes an ECDSA P256 private key into the
```

Comment:
```suggestion
// marshalX962UncompresedPrivateKey encodes an P-256 private key into the
```

I think it's more correct to say just P-256, since we're not using these for ECDSA. And I believe P-256 is the official name, as opposed to P256.

Reply:
Edited comment to catch another typo.

Reply:
Good point about us not using these keys for DSA. I updated a number of comments and variables in this file accordingly.